src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
```

Comment:
reset margins and paddings on * selector

Reply:
moved margins and paddings to * selector